docs/api-docs/management-apis/email-templates/email-template-examples.md

```diff
@@ -0,0 +1,116 @@
+
+
+## Adding a button
+
+To add a button to an email, you will need to add additional code to your template. Here is example HTML code that will add a blue button with white text to the account created template.
+
+Go to **Transactional Emails> Email Templates**. Click **...** next to **Account Created** and select **Edit Template**. Copy and paste the contents below into the Content editor. Ensure you paste the text outside the `<table>` tags. 
+  
+```html
+<table class="row">
+  <tr>
+    <th class="column">
+       <table>
+          <tr>
+             <th>
+                <a href="{{store.path}}" style = "background-color:blue; color:white" class="sigh-in">{{lang 'sign_in'}}</a>
+             </th>
+             <th class="expander"></th>
+          </tr>
+      </table>
+  </th>
+ </tr>
+</table>
+```
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+  
+  ### Note 
+  You can select a different button background and text color. We suggest adhering to the minimum color contrast standards set by [WCAG](https://webaim.org/articles/contrast/). 
+</div>
+</div>
+</div> 
+
+![Add button](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/email-templates-add-button.png "Button with white text on a blue background")
+
+## Changing logo size
+
+To change the size of your logo that appears in your email, specify width and height values in the Content editor. Go to **Transactional Emails> Email Templates**. Click **...** next to **Order Email** and select **Edit Template**. Go to the **Code** tab and replace: 
+
+```html
+<img src="{{store.logo.url}}" alt="{{store.logo.title}}">
+```
+with
+```html
+<img src="{{store.logo.url}}" alt="{{store.logo.title}}" width="200" height="200">
+```
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+  
+  ### Note  
+  You can also add width and height values to images.
+</div>
+</div>
+</div> 
+  
+![Change logo size](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/email-templates-change-logo-size.png "Change logo size")
+
+## Changing font size
+  
+To change the email font size, you will need to use the style attribute. You can define attributes such as color and font size within the paragraph `<p>` tag. Go to **Transactional Emails> Email Templates**, click **...** next to **Guest Account**, and select **Edit Template**. Go to the **Code** tab and replace: 
+  
+```html
+<p>{{lang 'help'}}</p>
+  ```
+ with 
+  
+  ```html
+<p style="font-size:16px;">{{lang 'help'}}</p>
+   ```
+To make your text responsive so that it can viewed on a variety of devices and screen sizes, you can change the text to the following:
```

Comment:
```suggestion
To make your text responsive so that it can viewed on a variety of devices and screen sizes, use viewport width (`vw`) unit to set the `font-size`.
```